src/main.rs

```diff
@@ -133,8 +133,10 @@ async fn main() {
                     }
                 }
                 Ok(Sk8brdMsgs::MsgListDevices) => list_device(&buf, msg.len),
-                Ok(m) => println!("uknown message type {m:?}"),
-                Err(e) => println!("got error '{e}' while processing msg: {msg:?}"),
+                Ok(m) => writeln!(stdout(), "unknown message type {m:?}").unwrap(),
```

Comment:
Could you please clarify, what's the difference between `println!()` and `writeln!(stdout(),)`

Reply:
`println!` is unreliable with raw mode tty